devito/dle/rewriters.py

```diff
@@ -377,12 +380,15 @@ def _simdize(self, iet):
         return processed, {}
 
     @dle_pass
-    def _node_parallelize(self, iet):
+    def _parallelize_shm(self, iet):
         """
-        Add OpenMP pragmas to the Iteration/Expression tree to emit shared-memory
-        parallel code.
+        Add OpenMP pragmas to the Iteration/Expression tree to emit SIMD and
+        shared-memory parallel code.
         """
-        return self._node_parallelizer.make_parallel(iet)
+        if self.params['openmp']:
+            return self._parallelizer_shm.make_parallel(iet)
+        else:
+            return iet, {}
```

Comment:
I think this is what make the test fail as with this (more rigorous) update, no pragmas are generated at all without openmp now. Not sure if you wanna go back to "serial pragmas" or add a "skip" openmp.